src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -51,61 +52,64 @@ object Lorre extends App with LazyLogging {
           noOp
         _ <-
         if (iteration % purgeAccountsInterval == 0)
-          TezosDatabaseOperations.purgeOldAccounts()
+          TezosDb.purgeOldAccounts()
         else
           noOp
     } yield ()
+
     Await.ready(processing, atMost = Duration.Inf)
     logger.info("Taking a nap")
     Thread.sleep(sleepIntervalInSeconds * 1000)
     mainLoop(iteration + 1)
   }
 
+  sys.addShutdownHook{
```

Comment:
Is this block in conflict with #154?

Reply:
Yes sure, will need to properly merge.

I couldn't keep track anymore of where each change was made, so duplication crops out here and there.